playground/mongo/Mongo.ApiService/Program.cs

```diff
@@ -12,6 +12,7 @@
 
 var app = builder.Build();
 
+app.MapDefaultEndpoints();
```

Comment:
It's actually interesting that we weren't calling this in the past but we were Adding Service Defaults. Do you happen to know why? Why do we need it now?

Reply:
We need it to get the `/alive`, and `/health` endpoints being used by the tests. When running it manually, it wasn't useful I'm guessing.